### ISSUE TITLE
Update a Customer

### ISSUE THREAD

**Issue by joseym** [balanced/balanced-node#25]
> package version: `1.0.5`

The [docs](https://docs.balancedpayments.com/1.1/api/customers/#update-a-customer) show this:

``` javascript
balanced.get('/customers/CU4ZLxuKnxGv1XulAOL4aTXq').set('name', 'alan turing').set('email', '123@example.com').save();
```

However I am unable to get that to work at all.

I'm achieving my update from the promise like so:

``` javascript
balanced.get('/customers/CU4ZLxuKnxGv1XulAOL4aTXq').then(
  function Success(customer){
    customer.name = 'alan turing';
    customer.email = '123@example.com';
    customer.save()
  }, 
  function Error(err){}
);
```

So I guess my question is: Are the docs wrong, or am I missing something?

**Comment by mjallday** [comment]
The [tests for the client library](https://github.com/balanced/balanced-node/blob/rev1/test/test.js#L68) uses your syntax. Also, [some of the examples](https://docs.balancedpayments.com/1.1/api/bank-accounts/#update-a-bank-account) in the docs use your syntax too.

@matthewfl can you take a look?

**Comment by matthewfl** [comment]
https://github.com/balanced/balanced-node/blob/rev1/lib/balanced.js#L77

both methods should work and are equally good.  I am not sure atm why .set method wasn't working.  Are you seeing an error when you try using the `.set` method on the promise?

**Comment by mjallday** [comment]
@matthewfl are there test cases for `.set`?

**Comment by matthewfl** [comment]
looks like this might be an issue when doing set without a `.` so something like `set('email` would fail while `set('links.customer` works

**Comment by matthewfl** [comment]
found this issue, fix will be out shortly

**Comment by matthewfl** [comment]
@joseym this is fixed now in version `1.0.6`

https://github.com/matthewfl/jsonapi-node/commit/2b5a25a12eb7c108a37a595cf3c4168e4a9e5b68

https://github.com/balanced/balanced-node/commit/89edcce6dc971726b459251e691e4f492557cb2e#diff-c1129c8b045390789fa8ff62f2c6b4a9R75

**Comment by joseym** [comment]
Excellent, thank you.
